content/en/docs/distributions/gke/deploy/upgrade.md

```diff
@@ -258,6 +260,8 @@ Kubeflow on Google Cloud doesn't guarantee the upgrade for each Kubeflow compone
 
 ### Upgrade Kubeflow cluster to v1.5
 
+Starting from Kubeflow v1.5.1 we upgraded ASM to v1.13. Please, follow the instructions on how to [upgrade ASM (Anthos Service Mesh)](#upgrade-asm-anthos-service-mesh). If you want to use ASM version prior to 1.11, refer to [the legacy instructions](https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/common/asm/deprecated/README.md).
```

Comment:
Using the term `Please` here shows politeness but audiences are looking for instruction but not communication. We can omit the `Please` here and directly say `Follow the instructions ...`.

Reply:
Fixed